cmd/analyze.go

```diff
@@ -171,6 +177,7 @@ func getPackagePath(inputPath string) string {
 	// Adjust this according to your specific requirements
 	dirPath = strings.TrimPrefix(dirPath, "../")
 	dirPath = strings.TrimPrefix(dirPath, "./")
```

Comment:
This is strange, I would have used something around

https://pkg.go.dev/path/filepath#Clean

and/or

https://pkg.go.dev/path/filepath#Dir

Reply:
Did you try @alegrey91 ?